index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aprendiendo Flexbox</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <img src="images/Imagen2.png" alt="logo" width="111px" height="74px"> 
+        <div class="boton">≡</div>
+    </header>
+    <main>
+        <h1 class="caja miNombre">
+            Felipe <br> Haybar 
+        </h1>
+        <section class="saludo">
+            <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+         </section>    
```

Comment:
[REQUIRED CHANGES]

- Hey, according to @fherrerao's review and according to the project requirements the container should be a `section` tag and as @Feliverse stated about using one section, please kindly wrap the `h1` and the `p` in a section tag.

Reply:
Hi @zieeco 
Nice work, I replace the tag in the next commit
Thanks